integration/eth2network/eth2_network.go

```diff
@@ -390,6 +390,7 @@ func (n *Impl) gethStartNode(executionPort, networkPort, httpPort, wsPort int, d
 		"--allow-insecure-unlock", // allows to use personal accounts over http/ws
 		"--nodiscover",            // don't try and discover peers
 		"--ipcdisable",            // avoid geth erroring bc the ipc path is too long
+		"--verbosity", "1",
```

Comment:
I'm guessing this is ERROR level too but worth adding comment next to it to say what '1' means probably